device-types/Fortinet/FG-800D.yaml

```diff
@@ -0,0 +1,96 @@
+---
+manufacturer: Fortinet
+model: FortiGate 800D
+slug: fortinet-fg-800d
+part_number: FG-800D
+u_height: 1
+weight: 8.6
+weight_unit: kg
+airflow: front-to-rear
+is_full_depth: false
+front_image: true
+rear_image: true
+comments: '[FortiGate 800D Series Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/FortiGate_800D.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PWR1
+    type: iec-60320-c14
+    maximum_draw: 187
+    allocated_draw: 128
+  - name: PWR2
+    type: iec-60320-c14
+    maximum_draw: 187
+    allocated_draw: 128
```

Comment:
It appears that these PSUs are hot swappable. These should thus be modelled under `module-bays`, and then the associated PSU modules should be modelled in the module-types folder if they do not already exist there